src/main/java/com/aquarium/aquarium_backend/Repositories/FishRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.aquarium.aquarium_backend.Repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import com.aquarium.aquarium_backend.databaseTables.Fish;
+
+@Repository
+public interface FishRepository extends JpaRepository<Fish, Integer> {
+  @Query("Select fsh from Fish fsh where fsh.aquarium.aquariumId= :aquariumId")
```

Comment:
fsh instead of fish? :D

Reply:
Since SQL deosn't care about big and small letters Fish and fish may get confused in querry so using short fsh is safer